content/en/docs/architecture/step-registry.md

```diff
@@ -171,8 +171,8 @@ ref:
   The step runs with custom credentials injected.
 {{< / highlight >}}
 
-Note that access to read these secrets from the namespace configured must be granted separately from the configuration being added to a step.
-By default, only secrets in the `test-credentials` namespace will be available for mounting into test steps.
+**Note that access to read these secrets from the namespace configured must be granted separately from the configuration being added to a step.
+By default, only secrets in the `test-credentials` namespace will be available for mounting into test steps, by following the secret-mirroring [document](https://github.com/openshift/release/blob/master/core-services/secret-mirroring/README.md).**
```

Comment:
The sentence does not make sense. the _by following..._ does not bind to anything.

_"...test steps. Please follow the secret-mirroring doc to set up a custom secret in that namespace."_ or something similar?

Reply:
Thanks, updated.